File: ArithmeticOperations.py
import math

def arithmetic_operations(num1, num2):
    sum_result = num1 + num2
    difference_result = num1 - num2
    product_result = num1 * num2
    quotient_result = num1 / num2 if num2 != 0 else "Undefined (division by zero)"
    
    return {
        "Sum": sum_result,
        "Difference": difference_result,
        "Product": product_result,
        "Quotient": quotient_result
    }

result = arithmetic_operations(20, 15)
#change the numbers what you want .

print(result)

#Created by : Saad Almalki

File: Math-Programming/ArithmeticOperations.java
import java.util.Scanner;

public class ArithmeticOperations {
    public static void main(String[] args) {

        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter the first number: ");
        int num1 = scanner.nextInt();
        System.out.print("Enter the second number: ");
        int num2 = scanner.nextInt();
        int sum = num1 + num2;
        int difference = num1 - num2;
        int product = num1 * num2;
        double quotient = num1 / (double) num2;
        System.out.println("Sum: " + sum);
        System.out.println("Difference: " + difference);
        System.out.println("Product: " + product);
        System.out.println("Quotient: " + quotient);

        scanner.close();
    }
}

//Created by : Saad Almalki , Using : Java language & Eclipse IDE